src/AllArticles.jsx

```diff
@@ -1,20 +1,25 @@
-import ArticleThumbnail from "./Article-thumbnail"
+import ArticleThumbnail from "./Article-thumbnail";
+import { getAllArticles } from "../apis";
 
-import { useState } from "react"
+import { useEffect, useState } from "react";
 
 function AllArticles() {
-
-const [allArticles, setAllArticles] = useState([])
+    const [allArticles, setAllArticles] = useState([]);
     
+    useEffect(() => {
+      getAllArticles().then(({articles}) => {
+        setAllArticles(articles)
+      });
+    }, []);
 
+  return (
+    <div className="flex">
```

Comment:
we need some loading state to show the user that the data is loading and its not a problem with the website